examples/cryptorealtime/terraform-setup/main.tf

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright 2019 Google
+ * <p/>
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same comment here on the license; also, `autogen foo.tf` should work for you now.

Reply:
all JAVA and terraform files are updated https://github.com/GoogleCloudPlatform/professional-services/pull/191/commits/05ab1977716987c886d5bb7bb221ea4c72cdc1ee